cs/course.md

```diff
@@ -0,0 +1,96 @@
+Principy předmětu
+=================
+
+BI-3DT
+------
+
+-   učí členové [3Dprint Labu](http://3dprint.fit.cvut.cz/)
+-   vždy uvítáme připomínky
+-   naučíme kompletní workflow k 3D tisku
+    -   tvorba modelu → optimalizace / opravy → slicování → tisk → zpracování
+
+RepRap
+------
+
+-   učíme na tiskárnách RepRap, protože:
+    -   open-source software
+    -   open-source hardware
+    -   cenová dostupnost
+    -   rapidní vývoj  
+-   *více o projektu později*
+
+Průběh
+------
+
+-   **cvičení** -- zde v A1048
+-   **tisk semestrálních prací** -- v labu A1153 ve zkouškovém období
+-   harmonogram semestru je na stránce [Cvičení][eduxtutorials] na EDUXu
+-   účast v zásadě nepovinná:
+    -   nevyhodíme vás za to, že jste nepřišli
+    -   úlohy na cvičeních jsou ale za body
+
+Hodnocení
+---------
+
+-   **30 bodů**  Domácí parametrický úkol v programu OpenSCAD
+-   **30 bodů**  Práce v semestru na cvičeních 
+-   **40 bodů**  Semestrální práce 
+
+Informace na stránce [Hodnocení][eduxclassification] na EDUXu je vždy prioritní 
+oproti informacím uvedených zde (informujte nás při neshodě).
+
+Cvičení
+-------
+
+Kopírují workflow tisku.
+
+1.  **Tvorba modelů**: program OpenSCAD
+1.  **Optimalizace** a **opravy**: program netfabb Basic
+1.  **Slicování**: programy Slic3r (a KISSlicer)
+1.  **Tisk**: program OctoPrint
+1.  1 cvičení také **Programování**: ADMesh
+
+Domácí úkol
+-----------
+
+-   ověření schopnosti ovládat program OpenSCAD na úrovni mistra Yody
+-   odevzdávání na GitHub
+-   zadání ve 3. týdnu (snad)
+-   parametrický model s pevně daným rozhraním
+
+Semestrální práce
+-----------------
+
+-   komplexní úloha
+-   opět kopíruje známý workflow
+-   zadání zveřejníme později
+-   zakončeno tiskem:
+    -   zkouškové obdobní
+-   40 % [hodnocení][eduxclassification] předmětu
+
+Instalace programů
+------------------
+
+-   viz [Programy](apps.md)
+-   do příště za domácí úkol
+-   příští cvičení konzultace případných problémů
+
+Když se mi to nelíbí
+--------------------
+
+-   pokud se vám něco nelíbí
+    -   řekněte nám to hned, ne až v anketě
+-   pokud se vám celý předmět nelíbí
+    -   odhlaste si ho do konce 2. týdne
+    -   dáte šanci jiným
+
+Facebook
+--------
+
+Na Facebooku existuje stránka 
+[Nechceme vypadnout z BI-3DT](https///www.facebook.com/groups/bi3dt/), na které
+můžete konzultovat své problémy s ostatními studenty, případně i s námi
+(vyučující sami tuto stránku aktivně sledují, ale nespoléhejte na to).
+
+[eduxtutorials]: http://edux.fit.cvut.cz/courses/BI-3DT.1/tutorials
```

Comment:
Can we get rid of the `.1`? it redirects anyway.

Reply:
Sure.